examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -1546,9 +1546,13 @@
     }
    ],
    "source": [
+    "def as_ragged(inputs, targets):\n",
+    "    _as_ragged = mm.ListToRagged()\n",
+    "    return _as_ragged(inputs), targets\n",
+    "\n",
     "history = model_bi_lstm.fit(\n",
-    "    loader,\n",
```

Comment:
why is this as_ragged conversion required here?

Reply:
Not needed. Removed.